<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>786</width>
    <height>502</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="CustomerBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>701</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Customer Information</string>
    </property>
    <widget class="QTextEdit" name="nameInput">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string>Enter Name</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="streetInput">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>60</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Enter Street Address</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="AptEdit">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>60</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>apt (optional)</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="cityInput">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>60</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Enter City</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="postalCodeInput">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>60</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Enter Postal Code</string>
     </property>
    </widget>
    <widget class="QLabel" name="promptNameError">
     <property name="geometry">
      <rect>
       <x>17</x>
       <y>30</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Please Enter Name</string>
     </property>
    </widget>
    <widget class="QLabel" name="promptStreetError">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>30</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Please Enter The Address</string>
     </property>
    </widget>
    <widget class="QLabel" name="promptCityError">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>30</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Please Enter City</string>
     </property>
    </widget>
    <widget class="QLabel" name="promptPostalError">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>30</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Please Enter Postal Code</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>220</y>
      <width>691</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>Customize Your Pizza!</string>
    </property>
    <widget class="QPushButton" name="resetBtn">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>190</y>
       <width>100</width>
       <height>32</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Reset Pizza</string>
     </property>
    </widget>
    <widget class="QPushButton" name="cancelBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>100</width>
       <height>32</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:red;</string>
     </property>
     <property name="text">
      <string>Cancel Order</string>
     </property>
    </widget>
    <widget class="QPushButton" name="placeOrderBtn">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>190</y>
       <width>100</width>
       <height>32</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:green;</string>
     </property>
     <property name="text">
      <string>Place Order</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="selectPizzaSize">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>100</y>
       <width>541</width>
       <height>49</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="smallPizzaOption">
        <property name="text">
         <string>Small</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="mediumPizzaOption">
        <property name="text">
         <string>Medium</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="largePizzaOption">
        <property name="text">
         <string>Large</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Toppings:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="firstToppingSelection">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>131</width>
       <height>32</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="secondToppingSelection">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>20</y>
       <width>131</width>
       <height>32</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="lastToppingSelection">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>20</y>
       <width>131</width>
       <height>32</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>49</width>
       <height>16</height>
      </size>
     </property>
     <property name="text">
      <string>Size</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>30</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Workshop 3</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>786</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
